src/database/seeders/20200213061827-Room.js

```diff
@@ -1,38 +1,78 @@
 export default {
-  up: (queryInterface) => queryInterface.bulkInsert(
+  up: queryInterface => queryInterface.bulkInsert(
     'Rooms',
     [
       {
-        type: '2 bed room, 3 bed rooms',
+        wifi: 'free',
+        air_condition: true,
+        area: 3,
+        room_number: 1236,
+        bedroom_number: 2,
+        amenities: ['fhgjk', 'fghj'],
+        description: 'image',
         accommodation_id: 1,
-        booked: true,
+        booked: false,
+        image: [
+          'http://res.cloudinary.com/elvis-rugamba/image/upload/v1581862917/pdwwtoha6r0jcoldvfuj.png',
+          'http://res.cloudinary.com/elvis-rugamba/image/upload/v1581862924/umgonemdhtar3yf14lua.png'
+        ],
         createdAt: new Date(),
         updatedAt: new Date()
       },
       {
-        type: '2 bed room, 3 bed rooms',
+        wifi: 'no',
```

Comment:
i think you have many independent fields which should be under amenities eg. wifi, air_conditioner should be in the amenities array. You can make it an object if you want but that would be you limiting a user. I think a user(host-supplier) should be able to add in as many amenities as they like. So I think to make amenities just an array